x/rollapp/types/key_transfer_genesis.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"encoding/binary"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ binary.ByteOrder
+
+const (
+	TransferGenesisMapKeyPrefix = "TransferGenesis/map/value/"
```

Comment:
not sure why u need this prefix

Reply:
Guess to make sure there is no collision with other store keys?
All the other store access use this pattern